doc/nrf/ug_matter_gs_ecosystem_compatibility_testing.rst

```diff
@@ -0,0 +1,56 @@
+.. _ug_matter_gs_ecosystem_compatibility_testing:
+
+Testing Matter ecosystem compatibility
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO:
+
+    * add info about SPECIFIC hubs and how to connect our DKs with them; we can link to the blog post and/or add links to the producer page
+    * add link to the ecosystems page in the getting started
+
+    * add information about mobile controllers to Testing page, as per comment in https://github.com/nrfconnect/sdk-nrf/pull/10189/files#
+
+Nordic Semiconductor's Matter platform and Matter applications integrated in the |NCS| are follow the most recent versions of the Matter protocol and are interoperable with several external Matter ecosystems.
+This interoperability relies on Matter's :ref:`multiple fabrics feature <ug_matter_overview_multi_fabrics>` and leverages Matter's support for using different wireless protocols for different use cases.
+
+You can test the interoperability of the Matter solution in the |NCS| with a selection of compatible Matter ecosystems by following the `Testing the nRF Connect platform with Apple, Google and Samsung ecosystems`_ tutorial on DevZone.
+This will allow you to set up and test the following scenario with Apple, Google, and Samsung ecosystems.
+
+.. figure:: images/matter_ecosystem_compatibility.png
```

Comment:
Do you think that the reader will understand what's going on with this image? In the blog those circles were used on multiple images to show evolving scope of specific ecosystems regarding Matter accessories.

Reply:
Also it shows specifically the Matter light bulbs, so maybe we should create something more generic to not make someone thinking that only light bulbs are supported.